tests/yast2_gui/yast2_hostnames.pm

```diff
@@ -22,9 +22,25 @@ sub run() {
     my $self   = shift;
     my $module = "host";
 
+    select_console 'root-console';
+    #	add 1 entry to /etc/hosts and edit it later
+    script_run "echo '80.92.65.530    n-tv.de ntv' >> /etc/hosts";
```

Comment:
invalid IP ?

Reply:
should be 80.92.65.53

Reply:
PR: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5424

Thanks!